docs/release/old-release-removal.md

```diff
@@ -0,0 +1,90 @@
+Old Release Series Removal Plan
+===============================
+
+In order to reduce clutter and disk usage on our repositories and build system,
+we should remove older OSG Software release series.  This will result in
+packages from those series becoming unavailable both to users, and to developers.
+
+
+Policy
+------
+
+We should delete a release series only when needing to download a package from
+that series is improbable.  I propose the following rule:
+
+Delete a release series when the _following_ series is completely out of
```

Comment:
> We will remove a release series when...

Since the document is titled "Old Release Series Removal Plan", you should stick to "removal" instead of "deletion" when talking about the high-level plan, imo.

Reply:
I wanted to avoid too much repetition, but sure, I can do that.